pkg/controller/telemetry/utils.go

```diff
@@ -168,6 +168,18 @@ var (
 			Help: "The total number of TCP connections failed to a workload.",
 		}, workloadLabels)
 
+	tcpConnectionTotalRetransInWorkload = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kmesh_tcp_workload_connections_total_retrans",
+			Help: "The total number of TCP connections retried to a workload.",
```

Comment:
As far as I understand, this metric describes the number of packet retransmission in a single connection, but this help message makes me feel it describes the number of connections retries sent to a workloads. 

One is the packet granularity and the other is the connection granularity.

Reply:
This metric adds up the number of `total_retrans` for every time `src` connect to `des`.

Reply:
+1  confusing to me